base/applications/notepad/rsrc.rc

```diff
@@ -1,148 +1,154 @@
-/*
```

Comment:
Please revert this change, and be sure that *no encoding change* happens there.
(in particular, don't use github editor to work on translations.)